runtime/src/lib.rs

```diff
@@ -283,8 +283,12 @@ impl pallet_nft::Config for Runtime {
 	type RuntimeEvent = RuntimeEvent;
 	type WeightInfo = ();
 	type RoleManager = Roles;
+	type Balance = Balance;
+	type Bank = Bank;
 	type EnsureGovernance = traits::SuccessOrigin<Runtime>;
 	type MaxSize = ConstU32<1000>;
+	type Fee = ConstU128<CENT>;
```

Comment:
should allow higher MaxSize
and also use higher Fee (at least a dollar)
Rename Fee to something more clear - PodFee